src/bundle.rs

```diff
@@ -117,6 +149,14 @@ impl Flags {
         self.outputs_enabled
     }
 
+    /// Flag denoting whether ZSA transaction is enabled.
+    ///
+    /// If `false`,  all notes within [`Action`]s in the transaction's [`Bundle`] are
+    /// guaranteed to be notes with native asset.
```

Comment:
_From str4d ([r2585544143](https://github.com/zcash/orchard/pull/471#discussion_r2585544143))_

```suggestion
    /// Flag denoting whether ZSA functionality is enabled in the transaction.
    ///
    /// If `false`, all notes within [`Action`]s in the transaction's [`Bundle`] are
    /// guaranteed to be notes with native asset. If `true`, `Action`s may use any asset.
```

Reply:
FIXED